lib/texts.js

```diff
@@ -109,6 +109,8 @@ ${leaves}`,
 	'tl.finishedWithErrors': () => 'Some executions finished with errors',
 	'tl.finishedWithSuccess': () => 'All executions finished successfully',
 	'tl.promptPassword': () => 'Please enter your passwod: ',
+	'tl.failedToCreateDir': () => 'Unknown error occurred while creating a new directory',
+	'tl.startLogRecording': (dir, device) => `Logging started on ${device}\nYou can find logs in ${dir}/${device}.log.`,
```

Comment:
Please adjust this message for windows platform

Reply:
I.e. replace "\n" with os.EOL and "/" with path.sep